pymc_marketing/mmm/delayed_saturated_mmm.py

```diff
@@ -342,9 +378,52 @@ def build_model(
                 dims="date",
             )
 
-            intercept = self.intercept_dist(
-                name="intercept", **self.model_config["intercept"]["kwargs"]
-            )
+            if self.time_varying_intercept or self.time_varying_media_effect:
+                time_index = pm.MutableData(
+                    "time_index",
+                    self._time_index,
+                    dims="date",
+                )
+
+            if self.time_varying_intercept:
+                tv_multiplier_intercept = time_varying_prior(
+                    name="tv_multiplier_intercept",
+                    X=time_index,
+                    X_mid=self._time_index_mid,
+                    positive=True,
+                    m=200,
+                    L=[self._time_index_mid + DAYS_IN_YEAR / self._time_resolution],
```

Comment:
I think it might be cleaner to use the `c` parameeer?
```
c: float
  The proportion extension factor. Used to construct L from X. Defined as S = max|X| such that X is in [-S, S]. 
  L is calculated as c * S. One of c or L must be provided. Further information can be found in Ruitort-Mayol et al.
```

Reply:
Unfortunately this causes some trouble with predicting out of sample. @bwengals did you find the cause for this?